core/src/main/java/org/mskcc/cbio/portal/scripts/MutationFilter.java

```diff
@@ -254,6 +235,18 @@ public int getRedactedRejects()
 	{
 		return this.redactedRejects;
 	}
+	
+   public Map<String, Integer> getMapRejections() {
+       return this.countMap;
+   }
+   
+   public void addRejectedVariant(Map<String, Integer> countMap, String mutation) {
+       if (this.countMap.get(mutation) != null) {
```

Comment:
I thought the `computeIfPresent` made the if / else unnecessary....is this not the case?